fake_filesystem_unittest_test.py

```diff
@@ -207,6 +207,29 @@ def testCopyRealFile(self):
             self.copyRealFile(real_file_path, fake_file_path,
                               create_missing_dirs=False)
 
+    def testCopyRealFileNoDestination(self):
```

Comment:
Here you are testing happens when you do not specify the destination path.  It is adequate to test only for the existence of the destination file.

The previous test tested all the details like the contents and stat.